### PR TITLE
vaapi_allocator: Wrap using RGB565 info #ifdef

### PR DESCRIPTION
**Pull request by Losev-Ivan**

### CODE REVIEW

File: _studio/shared/src/libmfx_allocator_vaapi.cpp [hunk 1]
```diff
@@ -179,7 +179,9 @@ static bool isFourCCSupported(unsigned int va_fourcc)
         case VA_FOURCC_P208:
         case VA_FOURCC_P010:
         case VA_FOURCC_AYUV:
+#if defined (MFX_ENABLE_FOURCC_RGB565)
         case VA_FOURCC_RGB565:
+#endif // MFX_ENABLE_FOURCC_RGB565
```
Comment (DenWolf) [inline _studio/shared/src/libmfx_allocator_vaapi.cpp]: Comment "// MFX_ENABLE_FOURCC_RGB565" is not needed

Comment (Losev-Ivan) [reply _studio/shared/src/libmfx_allocator_vaapi.cpp]: Done

File: _studio/shared/src/libmfx_allocator_vaapi.cpp [hunk 2]
```diff
@@ -71,10 +71,10 @@ unsigned int ConvertMfxFourccToVAFormat(mfxU32 fourcc)
         return VA_FOURCC_YV12;
     case MFX_FOURCC_AYUV:
         return VA_FOURCC_AYUV;
-#if defined (MFX_ENABLE_FOURCC_RGB565)
+#if (MFX_VERSION >= 1028)
```
Comment (mgonchar) [inline _studio/shared/src/libmfx_allocator_vaapi.cpp]: No, dedicated define ```MFX_ENABLE_FOURCC_RGB565``` is much better since we have it

Comment (Losev-Ivan) [reply _studio/shared/src/libmfx_allocator_vaapi.cpp]: Done

**Pull request merged**